### ISSUE TITLE
Add uppercase letters in bot name handling

### ISSUE THREAD

**Issue by enzo-bonnot** [Solidras/QuotesBot#3]
Currently when you add a character only the first letter will be in uppercase whether you put some in the middle of the name or not. I think it could be a great idea to be able to have uppercase letters in the middle of the character's name.

**Comment by Solidras** [comment]
Indeed, that's could be a good idea. I tried a few days ago, but I didn't have the time to fix this issue. Thanks to bring it up.
